src/test/com/gd/twitterstreamingtToDatalake/StreamingTweetsJobTest.scala

```diff
@@ -0,0 +1,28 @@
+package com.gd.twitterstreamingtToDatalake
+
+import org.scalatest.FunSuite
+
+
+class StreamingTweetsJobTest extends FunSuite {
```

Comment:
Please check out this guide for naming conventions of Scala tests/specs: http://www.scalatest.org/user_guide/selecting_a_style

Reply:
Are you referring to the name of class or name of test cases ?
This class name was autogenerated by inteiiJ. Test cases names, I have fixed.

Reply:
> Are you referring to the name of class or name of test cases ?
 
to both.

> This class name was autogenerated by inteiiJ.

We should not rely on autogenerated stuff. 
Please follow the convention.